descriptors/ModuleDescriptor-template.json

```diff
@@ -36,7 +36,10 @@
         {
           "methods": ["PUT"],
           "pathPattern": "/invoice/invoices/{id}",
-          "permissionsRequired": ["invoice.invoices.item.put"]
+          "permissionsRequired": ["invoice.invoices.item.put"],
+          "modulePermissions": [
+            "invoice-storage.invoices.item.put"
+          ]
```

Comment:
it also requires "invoice-storage.invoices.item.get", as the invoice is being fetched for folioInvoiceNo

Reply:
Done. Thank you.